packages/model-viewer/src/three-components/Renderer.ts

```diff
@@ -246,8 +246,8 @@ export class Renderer extends EventDispatcher {
     this.avgFrameDuration =
         (HIGH_FRAME_DURATION_MS + LOW_FRAME_DURATION_MS) / 2;
 
-    const width = this.width / scale;
-    const height = this.height / scale;
+    const width = Math.floor(this.width / scale);
+    const height = Math.floor(this.height / scale);
```

Comment:
Ooh good spot, I missed that.

_Potentially_ it (lines ~475) is moot, because it's using Math.min() with this.canvas3D dimensions? (which I think get floored).

But not 100% sure and I can't really check thoroughly right now; I am on vacation and won't be able to investigate/test this properly until end of next week.

Reply:
Hi there, I have also applied floor() in the render function.

In my specific testing (my real-world use case with fractional pixels values) I observed no difference here between using ceil() and floor() at all; they were always exactly the same as the (integer) this.canvas3D.width / height values.  (it's possible this Math.min() logic is no longer actually needed at all? but I don't know the code well enough to have any confidence there)

Anyway, for immediate purposes it seems prudent to be consistent and use floor here.

Reply:
Oh interesting, thanks for explaining.

I have created a local test page displaying 3 different models at different sizes (and with a mix of integer/floats for container sizes).

It appears to be working as per your above description; the <canvas> elements take the max width/height across all instances, and actual renders appear at correct dimensions.